database/app/src/main/java/com/google/firebase/referencecode/database/KotlinActivity.kt

```diff
@@ -0,0 +1,49 @@
+package com.google.firebase.referencecode.database
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.FirebaseDatabase
+import com.google.firebase.database.ValueEventListener
+import com.google.firebase.referencecode.database.interfaces.ActivityInterface
+
+class KotlinActivity : AppCompatActivity(), ActivityInterface {
+
+    private val TAG = "KotlinActivity"
```

Comment:
nit: doesn't matter here because it won;'t be shown in the actual snippets, but I think most people put `TAG` in the companion object in kotlin.